lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -732,57 +733,98 @@ def _get_default_parameters(kymo, channel):
 
     return {
         "pixel_threshold": KymotrackerParameter(
-            "Threshold",
-            "Set the pixel threshold.",
+            "Minimum intensity",
+            "Set the intensity threshold.",
             "int",
             max(1, int(np.percentile(data.flatten(), 98))),
             *(1, max(2, np.max(data))),
             True,
+            r"Minimum pixel intensity defines how many photons are required in a single pixel for "
+            r"something to be considered part of a track. This parameter should be chosen slightly "
+            r"above the baseline photon count. Higher values reject more noise, but actual peaks "
+            r"that should be part of the track may be missed.",
         ),
-        "track_width": KymotrackerParameter(
-            "Track width",
-            "Estimated spot width.",
+        "sigma": KymotrackerParameter(
+            "Search range",
+            "How much does the track fluctuate?",
             "float",
-            4 * position_scale,
-            *(0.0, 15.0 * position_scale),
+            2 * position_scale,
+            *(1.0 * position_scale, 5.0 * position_scale),
             True,
+            "Maximum search range defines how much the position of a particle is allowed to "
+            "fluctuate from one time point to the next while still being considered part of the "
+            "same track. Larger values will result in a wider range in which points are added to "
+            "a track.",
         ),
         "window": KymotrackerParameter(
-            "Window", "How many frames can a track disappear.", "int", 4, *(1, 15), True
+            "Maximum gap duration",
+            "How many frames can a track disappear.",
+            "int",
+            4,
+            *(1, 15),
+            True,
+            r"Maximum gap duration defines how big the gap between two points can be for them to "
+            r"connect. It should be chosen such that small gaps in a track can be overcome and "
+            r"tracked as one, but not so large that separate tracks are strung together.",
```

Comment:
```suggestion
            r"Maximum gap duration defines how many scan lines without detected points can occur "
            r"between two points and still be connected within a single track.  This can occur, for instance, "
            r"due to fluorophore blinking or the signal falling slightly below the intensity threshold. "
            r"This value should be chosen such that small gaps in a track can be overcome and "
            r"tracked as one, but not so large that separate tracks are strung together.",
```

Reply:
👍